deploy/gk-deploy

```diff
@@ -321,15 +324,15 @@ if [[ ${ABORT} -eq 1 ]]; then
 fi
 
 if [[ "${CLI}" == *oc* ]]; then
-  ${CLI} create -f ${TEMPLATES}/deploy-heketi-template.yaml
-  ${CLI} create -f ${TEMPLATES}/heketi-service-account.yaml
-  ${CLI} create -f ${TEMPLATES}/heketi-template.yaml
+  ${CLI} create -f ${TEMPLATES}/deploy-heketi-template.yaml  || true
```

Comment:
why is this  `|| true` ?
i.e. why do we want to ignore errors from the ${CLI} create commands?